x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -109,16 +88,7 @@ export function StickyTransactionProperties({
       }),
       fieldName: TRANSACTION_RESULT,
       val: idx(transaction, _ => _.transaction.result) || NOT_AVAILABLE_LABEL,
-      width: '25%'
-    },
-    {
-      label: i18n.translate('xpack.apm.transactionDetails.userIdLabel', {
-        defaultMessage: 'User ID'
-      }),
-      fieldName: USER_ID,
-      val: idx(transaction, _ => _.user.id) || NOT_AVAILABLE_LABEL,
-      truncated: true,
-      width: '25%'
+      width: '10%'
```

Comment:
Sorry, I tested it out locally and felt that 10% was probably too little space on smaller screens for the result to render on one line (example in your screenshot in the description). Can we change this to `14%`?